### PR TITLE
Kommander 1.1.0 Release Notes (Beta 0 - RC 1)

### PR DESCRIPTION
**Pull request by juliangieseke**
JIRA Based Release Notes for existing Kommander Releases. This can go in as soon as its ✅ 

## Jira Ticket
Before creating this pull request, please make sure you have a separate ticket for the docs team to track their work to review and merge, even for minor edits. If you need assistance, ask in the #doc-team channel on slack.

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `staging`, not `master`. 
- [ ] Provide an estimated date for deploying the doc change. Note: Improvements or fixes can be merged ASAP. 
- [ ] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

File: pages/ksphere/kommander/1.1.0-beta/release-notes/index.md [hunk 1]
```diff
@@ -11,56 +11,208 @@ enterprise: false
...
-- kommander-thanos: 0.1.13
-- grafana: 4.5.1
+-->
 
-### Version v1.1.0-beta.0 - April 8 March 2020
+### Version v1.1.0-rc.1 - May 29th 2020
+
+| Kubernetes Support | Version |
+| ------------------ | ------- |
+| **Minimum**        | 1.16.0  |
```
Comment (msdolbey) [inline pages/ksphere/kommander/1.1.0-beta/release-notes/index.md:51]: I think we support the minimum version of 1.15.0, according to Jimmi and Dimitri we support current max n-2
```suggestion
| **Minimum**        | 1.15.0  |
```

**Comment by msdolbey** [comment]
@juliangieseke 
Do we want to have the newest stuff first/above the last release, rather than replacing? - sort of like here: 
https://docs.d2iq.com/ksphere/konvoy/1.4/release-notes/

File: pages/ksphere/kommander/1.1.0-beta/release-notes/index.md [hunk 2]
```diff
@@ -11,56 +11,208 @@ enterprise: false
...
+| **Maximum**        | 1.17.x  |
+| **Default**        | 1.17.3  |
+
+#### Features/Improvements
+
+- Added Access Control functionalities to the UI
+- Added Kubekost integration to the UI
+- Removed namespace suffix from projects and platform service names
+- UI now trims input values to remove leading, trailing and duplicate spaces
```
Comment (msdolbey) [inline pages/ksphere/kommander/1.1.0-beta/release-notes/index.md:60]: In general, do we like Oxford commas in places like this, @russurquhart1 ?

File: pages/ksphere/kommander/1.1.0-beta/release-notes/index.md [hunk 3]
```diff
@@ -11,56 +11,208 @@ enterprise: false
...
+- Kommander now federates Prometheus operator, prometheus alert manager and karma to managed non-konvoy clusters.
+- improved aws tags to include cluster name
```
Comment (msdolbey) [inline pages/ksphere/kommander/1.1.0-beta/release-notes/index.md:62]: Totally a nit-picky thing, but AWS should be caps, yeah?
```suggestion
- Improved AWS tags to include cluster name
```

File: pages/ksphere/kommander/1.1.0-beta/release-notes/index.md (hunk 1 above)
Comment (juliangieseke) [reply pages/ksphere/kommander/1.1.0-beta/release-notes/index.md:51]: I guess this is a bit unclear… we might need to state 1.15, but it’s technically not supported by kudo, which is part of kommander. 🤷‍♂️
@jaxesn maybe? Can we drop 1.15 here or should we keep it?

**Comment by juliangieseke** [comment]
@msdolbey no, its supposed to be a historic view of all 1.1.x releases. But seems like you figured that already 👍🏻

File: pages/ksphere/kommander/1.1.0-beta/release-notes/index.md (hunk 2 above)
Comment (juliangieseke) [reply pages/ksphere/kommander/1.1.0-beta/release-notes/index.md:60]: hehe, I guess grammar/typos are generally a thing when I'm writing something 🤷

File: pages/ksphere/kommander/1.1.0-beta/release-notes/index.md (hunk 1 above)
Comment (juliangieseke) [reply pages/ksphere/kommander/1.1.0-beta/release-notes/index.md]: lets keep it as is for now…

File: pages/ksphere/kommander/1.1.0-beta/release-notes/index.md (hunk 2 above)
Comment (russurquhart1) [reply pages/ksphere/kommander/1.1.0-beta/release-notes/index.md:60]: If i see them, then I add them!

File: pages/ksphere/kommander/1.1.0-beta/release-notes/index.md [hunk 4]
```diff
@@ -11,56 +11,208 @@ enterprise: false
...
+| **Minimum**        | 1.15.0  |
+| **Maximum**        | 1.17.x  |
+| **Default**        | 1.17.3  |
+
+#### Features/Improvements
+
+- Added Access Control functionalities to the UI
+- Added Kubekost integration to the UI
+- Removed namespace suffix from projects and platform service names
+- UI now trims input values to remove leading, trailing and duplicate spaces
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1.0-beta/release-notes/index.md:60]:
```suggestion
- UI now trims input values to remove leading, trailing, and duplicate spaces
```

File: pages/ksphere/kommander/1.1.0-beta/release-notes/index.md [hunk 5]
```diff
@@ -11,56 +11,208 @@ enterprise: false
...
+- Kommander now federates Prometheus operator, prometheus alert manager and karma to managed non-konvoy clusters.
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1.0-beta/release-notes/index.md:61]:
```suggestion
- Kommander now federates Prometheus operator, prometheus alert manager, and karma to managed non-konvoy clusters.
```

File: pages/ksphere/kommander/1.1.0-beta/release-notes/index.md [hunk 6]
```diff
@@ -11,56 +11,208 @@ enterprise: false
...
+- Improved AWS tags to include cluster name
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1.0-beta/release-notes/index.md:62]:
```suggestion
- Improved AWS tags to include cluster name.
```

File: pages/ksphere/kommander/1.1.0-beta/release-notes/index.md [hunk 7]
```diff
@@ -11,56 +11,208 @@ enterprise: false
...
+- removed old, unsupported versions from version selector in create cluster form
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1.0-beta/release-notes/index.md:63]:
```suggestion
- Removed old, unsupported versions from version selector in create cluster form.
```

File: pages/ksphere/kommander/1.1.0-beta/release-notes/index.md [hunk 8]
```diff
@@ -11,56 +11,208 @@ enterprise: false
...
+- Added a way to deploy generic KUDO services
+- Added quota support for projects
+- Kommander now federates logging stack to managed non-konvoy clusters
 
-- federate karma-proxy only to Konvoy clusters
-- simplify kommander's grafana dashboard job
+#### Bug Fixes
+
+- Fixed kubeconfig download for certain clusters/users
+- Fixed uninstallation from Kommander addon
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1.0-beta/release-notes/index.md:99]:
```suggestion
- Fixed uninstall from Kommander addon
```

File: pages/ksphere/kommander/1.1.0-beta/release-notes/index.md [hunk 9]
```diff
@@ -11,56 +11,208 @@ enterprise: false
...
+| **Minimum**        | 1.16.0  |
+| **Maximum**        | 1.16.x  |
+| **Default**        | 1.16.8  |
+
+#### Features/Improvements
+
+- Added Support for Kommander to attach the cluster it is running on as managed Cluster
+- Added ability to access the managed cluster's Kubernetes API with a valid management cluster token
+- Enabled License validation
+- simplify kommander's grafana dashboard job
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1.0-beta/release-notes/index.md:190]:
```suggestion
- Simplifed kommander's grafana dashboard job
```

File: pages/ksphere/kommander/1.1.0-beta/release-notes/index.md [hunk 10]
```diff
@@ -11,56 +11,208 @@ enterprise: false
...
+- Added loading indicators for cluster creation form
+- Added Cluster ID to UI for easier identification in other dashboard
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1.0-beta/release-notes/index.md:192]:
```suggestion
- Added Cluster ID to UI for easier identification in other dashboards
```

File: pages/ksphere/kommander/1.1.0-beta/release-notes/index.md [hunk 11]
```diff
@@ -11,56 +11,208 @@ enterprise: false
...
+- Displaying users Username in the UI
+- Added Cluster Overview Tab to Cluster Details Page in UI
+- Added a way to add clusters to projects based on labels
+- Added support for nonResourceUrls in Roles in the UI
+- Hide Kubeconfig download link for clusters where that config might not be available
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1.0-beta/release-notes/index.md:197]:
```suggestion
- Hid Kubeconfig download link for clusters where that config might not be available
```

File: pages/ksphere/kommander/1.1.0-beta/release-notes/index.md [hunk 12]
```diff
@@ -11,56 +11,208 @@ enterprise: false
...
+- Improved Cluster Status Visualisation in the UI
+
+#### Bug Fixes
+
+- Allow retrying deleting clusters after it failed for e.g. permission reasons
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1.0-beta/release-notes/index.md:202]:
```suggestion
- Allow deleting clusters retry after failing. For example when there are permission issues.
```

File: pages/ksphere/kommander/1.1.0-beta/release-notes/index.md [hunk 13]
```diff
@@ -11,56 +11,208 @@ enterprise: false
...
+- federate karma-proxy only to Konvoy clusters
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1.0-beta/release-notes/index.md:203]:
```suggestion
- Federate karma-proxy only to Konvoy clusters
```

File: pages/ksphere/kommander/1.1.0-beta/release-notes/index.md [hunk 14]
```diff
@@ -11,56 +11,208 @@ enterprise: false
...
+- Fixed UI not showing some error messages
+- Fixed UI not allowing user to change context when attaching cluster
+- Disabled "View Logs" Link in UI for managed cluster which dont run Kibana
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1.0-beta/release-notes/index.md:206]:
```suggestion
- Disabled "View Logs" Link in UI for managed clusters not running Kibana
```

File: pages/ksphere/kommander/1.1.0-beta/release-notes/index.md (hunk 2 above)
Comment (cmastr) [reply pages/ksphere/kommander/1.1.0-beta/release-notes/index.md:60]: plus one for Oxford comma!

**Comment by juliangieseke** [comment]
🎉  thanks for reviewing! @cmastr what does "ready to merge" mean in this case? will you merge it today? (I cant due to lack of permissions)

**Comment by cmastr** [comment]
> 🎉 thanks for reviewing! @cmastr what does "ready to merge" mean in this case? will you merge it today? (I cant due to lack of permissions)

Merging!

**Pull request merged**